pkg/expr/sql/db_test.go

```diff
@@ -205,6 +206,80 @@ func TestErrorsFromGoMySQLServerAreFlagged(t *testing.T) {
 	require.Contains(t, err.Error(), "error in go-mysql-server")
 }
 
+func TestFrameToSQLAndBack_JSONRoundtrip(t *testing.T) {
+	expectedFrame := &data.Frame{
+		RefID: "json_test",
+		Name:  "json_test",
+		Fields: []*data.Field{
+			data.NewField("id", nil, []*int64{p(int64(1)), p(int64(2))}),
+			data.NewField("payload", nil, []*json.RawMessage{
+				p(json.RawMessage(`{"foo":1}`)),
+				p(json.RawMessage(`{"bar":"baz"}`)),
+			}),
```

Comment:
None of these values are null. So I wouldn't bother making them nullable personally.

```suggestion
			data.NewField("id", nil, []int64{1, 2}),
			data.NewField("payload", nil, []json.RawMessage{
				json.RawMessage(`{"foo":1}`),
				json.RawMessage(`{"bar":"baz"}`),
			}),
```